### DISCUSSION TITLE
Quasar qtable grid view input field not displaying in mobile view

### DISCUSSION THREAD

**Discussion by arunima-nair** [quasarframework/quasar#13766] [General - Components / Directives / etc]
**_In Quasar qtable grid view input field not displaying_**
Mobile view
![image](https://user-images.githubusercontent.com/37262910/175398891-b39b35da-0bfb-4acb-8edd-365fe5ad17ff.png)
web view
![image](https://user-images.githubusercontent.com/37262910/175399017-3db00792-be2a-488e-8cb3-b56246efff33.png)
code below:

                <q-table :grid="$q.screen.xs"
               :hide-header="$q.screen.xs"
                  :data="chkList"
                  row-key="checkId"
                  class="sticky-header"
                  virtual-scroll
                  :hide-bottom="true"
                  style="height: 310px"
                  :rows-per-page-options="[0]"
                  flat
                  animated
                >
                  <div slot="header" style="display: contents">
                    <q-tr>
                      <q-th style="text-align: left !important"
                        ><label class="text-subtitle2">Check List</label></q-th
                      >
                      <q-th style="text-align: left !important"
                        ><label class="text-subtitle2"
                          >Y/N</label
                        ></q-th
                      >
                      <q-th style="text-align: left !important"
                        ><label class="text-subtitle2">Image</label></q-th
                      >
                      <q-th style="text-align: left !important"
                        ><label class="text-subtitle2">Remarks</label></q-th
                      >
                    </q-tr>
                  </div>
                  <template v-slot:body="props">
                    <q-tr :props="props">
                      <q-td
                        key="checkName"
                        style="text-align: left !important"
                        separator="cell"
                      >
                      <q-input borderless autogrow v-model=" props.row.checkName"
                      label="  " type="textarea" readonly/>

                      </q-td>
                      <q-td
                        style="text-align: left !important"
                        key="isFeasible"
                        :props="props"
                      >
                        <q-toggle
                          size="md"
                          checked-icon="check"
                          color="green"
                          unchecked-icon="clear"
                          false-value="N"
                          true-value="Y"
                          v-model="props.row.isFeasible"
                        />
                      </q-td>

                      <q-td key="image" :props="props">
                        <q-btn
                          align="left"
                          flat
                          round
                          color="primary"
                          icon="camera_alt"
                          dense
                          @click="captureImage(props.row.checkId)"
                        >
                        </q-btn>
                        <q-img
                          :src="'data:image/png;base64,' + props.row.image"
                        />
                      </q-td>
                      <q-td
                        style="text-align: left !important"
                        key="remark"
                        :props="props"
                      >
                        <q-input
                          dense
                          filled
                          style="font-size: 15px; text-align: left !important"
                          v-model="props.row.remark"

                        />
                      </q-td>
                    </q-tr>
                  </template>
                </q-table>
              ****

**Comment by metalsadman** [comment]
you didn't define the view for when it's grid using `items` slot. take idea from this example https://codepen.io/metalsadman/pen/ZgKexK
